mlir/test/Conversion/mqtopt-to-mqtref.mlir

```diff
@@ -13,7 +13,8 @@
 module {
     // CHECK-LABEL: func.func @testConvertAllocOpAttribute()
     func.func @testConvertAllocOpAttribute() {
-        // CHECK: %[[r_0:.*]] = "mqtref.allocQubitRegister"() <{size_attr = 2 : i64}>
+        // CHECK: %[[I2:.*]] = arith.constant 2 : index
```

Comment:
Similar to the QIR conversion, I would have expected this to be a statically defined memref.